src/commands/submit.ts

```diff
@@ -0,0 +1,25 @@
+import { Message, MessageEmbed, TextChannel } from 'discord.js';
+import { CHANNELS } from '@root/config';
+
+export const description = 'Submit a profile picture to the pfp contest.';
+export const extendedHelp = 'Exactly one file must be attached when running this command.';
+export const usage = '[more information]';
+
+export async function run(msg: Message, [imgDesc]: [string]): Promise<Message> {
+	if (msg.attachments.size !== 1) return msg.channel.send(extendedHelp);
```

Comment:
Do we think that this should be in an `argParser` function to stay in line with how we've been doing incorrect usage handling?